src/App.tsx

```diff
@@ -1,31 +1,46 @@
+import { Provider } from 'react-redux'
 import { RouterProvider, createBrowserRouter } from 'react-router-dom'
-import { ForgetPassword, Home, Login, Register, ResetPassword } from './Pages'
 import { AuthLayout, MasterLayout, NotFound } from './Components'
+import { AddNewRoom, Ads, Bookings, Facilities, ForgetPassword, Home, Login, Register, ResetPassword, Rooms, Users } from './Pages'
+import Store from './Redux/Store'
 
 
 
-  function App() {
-
-const routes = createBrowserRouter([
-  {
-    path: '/', element: <AuthLayout />, errorElement: <NotFound />, children: [
-      { index: true, element: <Login /> },
-      { path: "login", element: <Login /> },
-      { path: "register", element: <Register /> },
-      { path: "forgetPassword", element: <ForgetPassword /> },
-      { path: "resetPassword", element: <ResetPassword /> },
-    ]
-  },
-  {
-    path: 'dashboard', element: <MasterLayout />, errorElement: <NotFound />, children: [
-      { index: true, element: <Home /> },
-    ]
-  },
-])
-
-return <>
-  <RouterProvider router={routes} />
-</>
+function App() {
+
+  const routes = createBrowserRouter([
+
+    {
+      path: "dashboard", element: <MasterLayout />, errorElement: <NotFound />, children: [
+        { index: true, element: <Home /> },
+        { path: 'users', element: <Users /> },
+        { path: 'rooms', element: <Rooms /> },
+        { path: 'add-New-Room', element: <AddNewRoom /> },
```

Comment:
you can change this path to `rooms/add-new-room`, for better website hierarchy  and to indicate that we get to this page from the rooms page.